app/src/main/java/com/croconaut/tictactoe/model/Board.java

```diff
@@ -0,0 +1,200 @@
+package com.croconaut.tictactoe.model;
+
+
+import android.support.annotation.CheckResult;
+import android.support.annotation.IntRange;
+import android.support.annotation.NonNull;
+import android.support.annotation.RestrictTo;
+import android.support.annotation.VisibleForTesting;
+
+import static com.croconaut.tictactoe.utils.Assertions.assertIsInRange;
+import static com.croconaut.tictactoe.utils.Assertions.assertIsBiggerThan;
+import static com.croconaut.tictactoe.utils.Assertions.assertIsSeed;
+import static com.croconaut.tictactoe.utils.Assertions.assertNotNull;
+
+/**
+ * This class represents TicTacToe game playing field and its logic.
+ * <p>
+ * See also: {@link Cell}, {@link GameState}, {@link Seed}
+ */
+public final class Board {
```

Comment:
Missing reliable and also usable `toString` method. Consider to print using 2D array `mBoard`.